.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: somesy
+  name: somesy
+  description: Sync your metadata files with somesy
+  entry: somesy sync
+  language: python
+  always-run: true
```

Comment:
I get 

```
[WARNING] Unexpected key(s) present on https://github.com/Materials-Data-Science-and-Informatics/somesy => somesy: always-run
```

And also, shouldn't this be removed? (that was the reason for adding the `files` key)